emcc.py

```diff
@@ -3142,17 +3138,12 @@ def process_libraries(libs, lib_dirs, temp_files):
             logger.debug('found library "%s" at %s', lib, path)
             temp_files.append((i, path))
             consumed.append(i)
-            found = True
-            break
-        if found:
-          break
-      if found:
-        break
-    if not found:
-      jslibs = building.path_to_system_js_libraries(lib)
-      if jslibs:
-        libraries += [(i, jslib) for jslib in jslibs]
-        consumed.append(i)
+            continue
```

Comment:
It doesn't look like this `continue` does anything since it is at the end of a loop. Also, why is breaking when the lib is found no longer necessary?

Reply:
Oh crap you are right that code as needed :(